CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
3.6 is a very high CMake version to require. You aren't using any special features so I would go with something lower, maybe 3.0 since it added a bunch of nice stuff.